did/document.go

```diff
@@ -42,9 +42,9 @@ func (vms VerificationMethods) FindByID(id DID) *VerificationMethod {
 	return nil
 }
 
-// Remove removes a VerificationMethod from the slice.
-// If a verificationMethod was removed with the given DID, it will be returned
-func (vms *VerificationMethods) Remove(id DID) *VerificationMethod {
+// remove a VerificationMethod from the slice.
+// If a verificationMethod was removed with the given URI it will be returned or nil otherwise
+func (vms *VerificationMethods) remove(id DID) *VerificationMethod {
```

Comment:
why return it if no callers use it?

Reply:
To be on par but that doesn't make sense for a private method